### DISCUSSION TITLE
Sanchez on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by KG4JYA** [nullpainter/sanchez#101] [Q&A]
Hello All,

I have not been able to get Sanchez to work at all on a Raspberry Pi 4 using sanchez-v1.0.24-linux-arm.tar.gz. I am unclear as to the dependencies needed, can anyone advise?

Thanks

**Comment by nullpainter** [comment]
I haven't run Sanchez on a Pi for some time - does the issue raised at https://github.com/nullpainter/sanchez/issues/98 help at all? I know this is for an Orange Pi, but it may assist. I think a lot also depends on the distribution you're using. Certainly compiling from source is the easiest - it sounds intimidating, but it's remarkably straightforward with dotnet.

**Reply by KG4JYA** [reply]
I executed the suggestions in issue #98 and got it to install.
